release/ray_release/job_manager/anyscale_job_manager.py

```diff
@@ -325,7 +325,7 @@ def _get_logs():
                     )
                     print("", flush=True)
             output = buf.getvalue().strip()
-            if "### Starting ###" not in output:
+            if not output:
```

Comment:
could you add a comment on why/what this is doing? like why there are two ways to get `output`?

Reply:
Totally, I'll add to the PR before this one